src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCaseAboutToSubmitHandlerTest.java

```diff
@@ -264,4 +265,34 @@ void givenAdjournmentNextHearingIsPaper_thenUpdateHearingChannel() {
         assertThat(callback.getCaseDetails().getCaseData().getLatestHearing().getValue().getHearingChannel()).isEqualTo(HearingChannel.PAPER);
     }
 
+    @DisplayName("When adjournment is enabled and case hearing type is face_to_face and Adjournment next hearing type is Paper "
+        + ", then case hearing type should updated the wants to attend to no.")
+    @Test
+    void givenAdjournmentNextHearingIsPaper_thenUpdateWantsToAttend() {
+        HearingDetails hearingDetails = new HearingDetails();
+        hearingDetails.setHearingChannel(HearingChannel.FACE_TO_FACE);
+        callback.getCaseDetails().getCaseData().setHearings(Arrays.asList(new Hearing(hearingDetails)));
```

Comment:
callback.getCaseDetails().getCaseData() can be a local var

Reply:
done